### DISCUSSION TITLE
Compatible with Mageplaza_SocialShare

### DISCUSSION THREAD

**Discussion by pabodah** [breezefront/community#138] [3. Integrations]
The module was installed and seems to be working in Breeze environment.

1. There is a JS file added via the module - [https://static.addtoany.com/menu/page.js](https://static.addtoany.com/menu/page.js)
2. The above will load the share icons with a HREF embedded
3. Issue is when the product detail page is opened via the category detail page, the icons are missing the HREF therefore the icon click will not work
4. When the product above product detail page is refreshed, the icons will be loaded with HREF embedded and the icon click will be working
6. I tried with custom.js file as per [https://breezefront.com/docs/custom-javascript](https://breezefront.com/docs/custom-javascript) but it did not work
 `(function () {
    'use strict';

    $(document).on('breeze:load', function () {
        //adding the content in the JS file
    });
})();`
7.  Is there a way to load the above JS file to be loaded so the requirement fullfills?

**Comment by vovayatsyuk** [comment]
You need to call `a2a.init_all()` according to the [AddToAny docs](https://www.addtoany.com/buttons/api/#init):

```js
$(document).on('breeze:load', function () {
  a2a.init_all();
});
```
